lib/puppet_x/templates/dsc/init_ps.ps1

```diff
@@ -736,6 +736,8 @@ function Start-PipeServer
     $Encoding
   )
 
+  Add-Type -AssemblyName "System.Core, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"
```

Comment:
What about for systems that don't have the `3.5.0.0` version of `System.Core`?

Reply:
More commentary on https://github.com/puppetlabs/puppetlabs-powershell/pull/214#discussion_r177478151